gen3_util/files/uploader.py

```diff
@@ -42,15 +55,19 @@ def _upload_file_to_signed_url(file_name, md5sum, metadata, signed_url):
 
 
 def _update_indexd(attachment, bucket_name, document_reference, duplicate_check, index_client, md5sum, object_name,
-                   program, project, metadata=None):
+                   program, project, metadata=None, specimen_id=None, patient_id=None, task_id=None, observation_id=None):
     hashes = {'md5': md5sum}
     assert 'id' in document_reference, document_reference
     guid = document_reference['id']
     if metadata is None:
         metadata = {
             **{
```

Comment:
Ah, the `**` double stars! These aren't varargs are they?

Reply:
yes, above